27
[2, 7, 982, 929335, 34477]
-15
[2, 7, 982, 929335, 34477]
[6, 5, 58, 991, 42124]
-15
[6, 5, 58, 991, 42124]
[6, 5, 58, 991, 42124]
[6, 5, 58, 991, 42124]
[6, 5, 58, 991, 42124]
[6, 5, 58, 991, 42124]
Error: RuntimeError: Out of bounds memory access (evaluating 'instance.exports._start()')
Stack: python.wasm.wasm-function[_PyDict_Next]@[wasm code]
python.wasm.wasm-function[PyDict_Next]@[wasm code]
python.wasm.wasm-function[_PyUnicode_ClearInterned]@[wasm code]
python.wasm.wasm-function[finalize_interp_clear]@[wasm code]
python.wasm.wasm-function[_Py_Finalize]@[wasm code]
python.wasm.wasm-function[Py_FinalizeEx]@[wasm code]
python.wasm.wasm-function[Py_RunMain]@[wasm code]
python.wasm.wasm-function[pymain_main]@[wasm code]
python.wasm.wasm-function[Py_BytesMain]@[wasm code]
python.wasm.wasm-function[main]@[wasm code]
python.wasm.wasm-function[__main_void]@[wasm code]
python.wasm.wasm-function[_start]@[wasm code]
start@module-ready-wasi-py.js:498:32
@module-ready-wasi-py.js:5225:15
